docs/src/appendix/cheatcodes/start_elect.md

```diff
@@ -0,0 +1,47 @@
+# `start_elect`
+
+> `fn start_elect(contract_address: ContractAddress, sequencer_address: ContractAddress)`
+
+Changes the sequencer address for a contract at the given address.
+The change can be canceled with [`stop_elect`](./stop_elect.md).
+
+- `contract_address` - target contract address
```

Comment:
Actually does it make sense to implement it this way if we already have global implementations of syscalls? Shouldn't it take `CheatTarget` as well as the other cheatcodes?

Reply:
There's #1099, since the work on this has been started before the first PR with `CheatTarget` was merged. I'll probably update the whole PR to use `CheatTarget` once the tests for this are merged (#1118).

Reply:
Wouldn't it make additional tests needed?